src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/PossibleDeNovoUnitTest.java

```diff
@@ -30,105 +31,114 @@ public Object[][] deNovo() {
+        tests.add(new Object[]{g11gq30, g01gq30, g11gq30, false, false});
+        tests.add(new Object[]{g11gq30, g01gq30, gNo, false, false});
+
+        tests.add(new Object[]{g11gq30, g11gq30, g00gq30, false, false});
+        tests.add(new Object[]{g11gq30, g11gq30, g01gq30, false, false});
+        tests.add(new Object[]{g11gq30, g11gq30, g11gq30, false, false});
+        tests.add(new Object[]{g11gq30, g11gq30, gNo, false, false});
+
+        tests.add(new Object[]{g11gq30, gNo, g00gq30, false, false});
+        tests.add(new Object[]{g11gq30, gNo, g01gq30, false, false});
+        tests.add(new Object[]{g11gq30, gNo, g11gq30, false, false});
+        tests.add(new Object[]{g11gq30, gNo, gNo, false, false});
+
+
+        tests.add(new Object[]{gNo, g00gq30, g00gq30, false, false});
+        tests.add(new Object[]{gNo, g00gq30, g01gq30, false, false});
+        tests.add(new Object[]{gNo, g00gq30, g11gq30, false, false});
+        tests.add(new Object[]{gNo, g00gq30, gNo, false, false});
+
+        tests.add(new Object[]{gNo, g01gq30, g00gq30, false, false});
+        tests.add(new Object[]{gNo, g01gq30, g01gq30, false, false});
+        tests.add(new Object[]{gNo, g01gq30, g11gq30, false, false});
+        tests.add(new Object[]{gNo, g01gq30, gNo, false, false});
+
+        tests.add(new Object[]{gNo, g11gq30, g00gq30, false, false});
+        tests.add(new Object[]{gNo, g11gq30, g01gq30, false, false});
+        tests.add(new Object[]{gNo, g11gq30, g11gq30, false, false});
+        tests.add(new Object[]{gNo, g11gq30, gNo, false, false});
+
+        tests.add(new Object[]{gNo, gNo, g00gq30, false, false});
+        tests.add(new Object[]{gNo, gNo, g01gq30, false, false});
+        tests.add(new Object[]{gNo, gNo, g11gq30, false, false});
+        tests.add(new Object[]{gNo, gNo, gNo, false, false});
 
         final Genotype g00NoPl = new GenotypeBuilder("2", homRef).DP(10).GQ(GQ30).make();
         final Genotype g01NoPl = new GenotypeBuilder("1", het).DP(10).GQ(GQ30).make();
         final Genotype g11NoPl = new GenotypeBuilder("3", homVar).DP(10).GQ(GQ30).make();
 
-        tests.add(new Object[]{g00NoPl, g01NoPl, g11NoPl, false});
-        tests.add(new Object[]{g00NoPl, g00NoPl, g01NoPl, true});
+        tests.add(new Object[]{g00NoPl, g01NoPl, g11NoPl, false, false});
+        tests.add(new Object[]{g00NoPl, g00NoPl, g01NoPl, true, false});
+
+        final Genotype g00gq45 = new GenotypeBuilder("2", homRef).DP(50).PL(new int[]{0,100,100}).AD(new int[]{10, 0}).GQ(GQ45).make();
+        final Genotype g01gq45 = new GenotypeBuilder("1", het).DP(50).PL(new int[]{100, 0, 100}).AD(new int[]{5, 5}).GQ(GQ45).make();
+        final Genotype g11gq45 = new GenotypeBuilder("3", homVar).DP(50).PL(new int[]{100, 100, 0}).AD(new int[]{0, 10}).GQ(GQ45).make();
+
+        tests.add(new Object[]{g00gq45, g00gq45, g00gq45, false, false});
+        tests.add(new Object[]{g00gq45, g00gq45, g01gq45, false, true});
```

Comment:
only one positive case for this second condition? A quick inline comment explaining these extra tests would be helpful.

Reply:
done